### ISSUE TITLE
Crew Application

### ISSUE THREAD

**Issue by midwesthonorflight** [shmakes/hf-basic#206]
We have been entering our crew in as Guardians to still be pulled in for the manifest.  Would there be a "Crew" option for applications to try and keep them in separate counts/areas?

**Comment by shmakes** [comment]
We normally just copy the flight crew from the previous flight.

**Comment by shmakes** [comment]
Not saying we can't do this. Just need some more info.

**Comment by midwesthonorflight** [comment]
Could there be another type of application for crew members?
![image](https://user-images.githubusercontent.com/40114348/76362908-f11ae980-62ef-11ea-8ec6-96dc6b5fe074.png)

When my leads are adding their medical/bus captains in, it makes it difficult for me to keep track of our Guardian count and to know which is supposed to be added as a Guardian and which was supposed to be left alone for Medical/Bus Captains/Media/etc.
![image](https://user-images.githubusercontent.com/40114348/76363006-27f0ff80-62f0-11ea-9a7b-7ab4dff0c05c.png)

Certainly don't want to make it more difficult than it has to be but if it is possible, I would appreciate a different application in a different color for keeping tabs on what's needed for Guardians.  Can still pull down in the Guardians if prefered or can just be in the "All" export alone since they won't be getting the mailings like the Veterans/Guardians do.

**Comment by bjbradl4** [comment]
Steve,  I believe a simplified Application is what he is requesting with Full Name (first, middle, last) birth date, sex, seat assignment, bus assignment, flight assignment, dates, email.  Maybe a place to type in role (ex: med team, bus captain, tour lead, DC Guardian, etc)

Basically, only the info needed for a flight manifest/Bus Assignment and also the NF option. So no apparel, vet assigned, emergency contacts.  I can't look at a database right now but I'll double check once I can and see if I need to add anything to the comment.

**Comment by midwesthonorflight** [comment]
Yes, just the basics that would still link in the flights but appear as a different type of application to keep seperate from the Guardians and Veterans.

**Comment by shmakes** [comment]
I will be looking into this soon. Thank you both for the feedback!
